mache-couchbase/src/main/java/com/excelian/mache/couchbase/builder/CouchbaseProvisioner.java

```diff
@@ -97,17 +67,27 @@ public static ClusterBuilder couchbase() {
      * A builder with defaults for a Couchbase cluster.
      */
     public static class CouchbaseProvisionerBuilder {
-        private final ConnectionContext<Cluster> connectionContext;
         private final BucketSettings bucketSettings;
+        private CouchbaseEnvironment couchbaseEnvironment = DefaultCouchbaseEnvironment.create();
```

Comment:
Let's only initialise if it to default if it is null when we need it.

Ask don't tell.